hnsecure/pom.xml

```diff
@@ -80,6 +80,10 @@
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-direct</artifactId>
     </dependency>
+    <dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-netty</artifactId> 
+</dependency>
```

Comment:
I'm not too sure where netty is used. Was this required in the end?

Reply:
Rtrans endpoint expects a tcp connection.